### ISSUE TITLE
how long it took you to train using MSCOCO and WIKIArt

### ISSUE THREAD

**Issue by anmui** [AlienZhang1996/S2WAT#1]
@AlienZhang1996 
hi,is a good job,I want to know how long it took you to train using MSCOCO and WIKIArt?

Does it need to train 40000 epoch?

Thanks!😊

**Comment by AlienZhang1996** [comment]
Hi,

It takes me about 10 hours to train S2WAT for 40000 iterations on a single
V100. Actually, 40000 is an empirical value that I found in experiments
under the settings (batch size, warm up, learning rate, etc) in code. Lower
losses may appear if you train with more iterations.

Best!

anmui ***@***.***> 于2023年3月5日周日 15:40写道：
